server/user/index.js

```diff
@@ -1,17 +1,18 @@
 'use strict';
 
-const auth = require('passport').authenticate('jwt');
+const { authenticate } = require('../shared/auth');
 const ctrl = require('./user.controller');
 const model = require('./user.model');
 const router = require('express').Router();
 
 router
   // Public routes:
-  .post('/users/login', ctrl.login)
+  .post('/users/login', authenticate('local'), ctrl.login)
   .post('/users/forgotPassword', ctrl.forgotPassword)
   .post('/users/resetPassword', ctrl.resetPassword)
```

Comment:
This route should validate the token. 
We're just checking whether the token exists in the database, but we don't validate token at all which makes audience, expiration and other JWT goodies we set on the token unused.
Create a middleware that verifies the JWT.
Also, after using the token it should be removed from the database to avoid the possibility to use it more than once.

Reply:
👍